sqlmesh/core/loader.py

```diff
@@ -182,9 +182,17 @@ def _load_metrics(self) -> UniqueKeyDict[str, MetricMeta]:
     def _load_external_models(self) -> UniqueKeyDict[str, Model]:
         models: UniqueKeyDict[str, Model] = UniqueKeyDict("models")
         for context_path, config in self._context.configs.items():
-            path = Path(context_path / c.SCHEMA_YAML)
+            schema_path = Path(context_path / c.SCHEMA_YAML)
+            external_models_path = context_path / c.EXTERNAL_MODELS
 
-            if path.exists():
+            paths_to_load = []
+            if schema_path.exists():
+                paths_to_load.append(schema_path)
+
+            if external_models_path.exists() and external_models_path.is_dir():
+                paths_to_load.extend(p for p in external_models_path.glob("*.yaml"))
```

Comment:
do you need to do an iterator here? doesn't glob just return a list? extend(x.glob())

Reply:
Oh, true. Have simplified it